SU-SWS/stanford_profile_helper
# D8CORE-4092: fixing up the padding to be on left and right

# READY FOR REVIEW

# Summary
- In a previous request, D8CORE-3673  and D8CORE-3163 adding padding to the left and right. When reviewing the code, the padding was top and bottom.
- I switched it to left and right.

# Needed By (Date)
- 4/25

# Urgency
- Med

# Steps to Test

1. Pull in this change.
2. Add a list with a single News card. Notice no padding top.
3. Notice there is padding on either side of the column.
4. Add three lists to a single row.
5. Notice they all align top.

# Affected Projects or Products
- Lists in Core
- Lists in SOE

# Associated Issues and/or People
- D8CORE-4092
- Previous related tickets: D8CORE-3673 & D8CORE-3163

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
